syntax = "proto3";

package greeter;

// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greeting.
message HelloReply {
  string message = 1;
}

// The greeting service definition.
service GreeterService {
  rpc SayHello (HelloRequest) returns (HelloReply);
}
